pages/instructorAssessmentSettings/instructorAssessmentSettings.js

```diff
@@ -40,15 +40,15 @@ router.get('/', function (req, res, next) {
     ],
     function (err) {
       if (ERR(err, next)) return;
-      debug('render page');
-      let host = config.serverCanonicalHost || 'https://' + req.headers.host;
-      res.locals.studentLink =
-        host +
+      const host = config.serverCanonicalHost || `${req.protocol}://${req.headers.host}`;
```

Comment:
Just to make sure I'm understanding, this use of `req.protocol` won't affect us in prod because we always set `serverCanonicalHost`, right?

Reply:
That's true, yes!